liquidctl/cli.py

```diff
@@ -264,13 +269,28 @@ def main():
 
     if args['--debug']:
         args['--verbose'] = True
-        logging.basicConfig(level=logging.DEBUG, format='[%(levelname)s] %(name)s: %(message)s')
-        _LOGGER.debug('running %s', _gen_version())
+        format_color='%(log_color)s[%(levelname)s]%(name)s%(reset)s: %(message)s'
```

Comment:
I think you may have accidentally consumed a space:
```diff
-[%(levelname)s] %(name)
+[%(levelname)s]%(name)
```

I think the former looks better, but let me know if you disagree and this was intentional.

Reply:
yup it looks like I did, I like the space more too, and I messed that up a few times during my initial testing with copy paste errors. Ill fix that though